ddtrace/llmobs/_writer.py

```diff
@@ -109,7 +109,7 @@ def periodic(self) -> None:
 
         data = self._data(events)
         try:
```

Comment:
this isn't actually covering any exception now, right? the `safe_json` can't raise. Should we do away with the `try`?

Reply:
Good catch, thanks!